src/nft/components/bag/BagFooter.tsx

```diff
@@ -286,16 +285,13 @@ export const BagFooter = ({
           </ActionButton>
         </TraceEvent>
       </Footer>
-      {showTokenSelector && (
-        <BagTokenSelectorModal
-          selectedCurrency={activeCurrency ?? undefined}
-          handleCurrencySelect={(currency: Currency | undefined) => {
-            setInputCurrency(currency)
-            toggleTokenSelector()
-          }}
-          overlayClick={toggleTokenSelector}
-        />
-      )}
+      <CurrencySearchModal
+        isOpen={tokenSelectorOpen}
+        onDismiss={() => setTokenSelectorOpen(false)}
+        onCurrencySelect={(currency: Currency) => setInputCurrency(currency.isNative ? undefined : currency)}
+        selectedCurrency={activeCurrency ?? undefined}
+        onlyShowCurrenciesWithBalance={true}
```

Comment:
I'm missing some expected search results such as $UNI with the search modal here vs the one used by the swap widget
<img width="464" alt="Screen Shot 2023-01-31 at 22 09 53 " src="https://user-images.githubusercontent.com/11512321/215965650-2aee6847-6151-41fa-9970-e2ab33509473.png">
<img width="458" alt="Screen Shot 2023-01-31 at 22 11 27 " src="https://user-images.githubusercontent.com/11512321/215965654-16078f0a-b079-45fe-87d7-587d6358b011.png">

Reply:
since we are only showing tokens with a balance I am just going to remove the searching alternative token lists feature